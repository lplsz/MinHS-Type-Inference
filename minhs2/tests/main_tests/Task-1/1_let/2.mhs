main
     = let x = 1;
       in let y = x;
              x = True;
          in y;
