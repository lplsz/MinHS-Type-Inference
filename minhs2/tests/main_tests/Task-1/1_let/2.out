main :: Int = let x :: Int = 1;
              in let y :: Int = x;
                     x :: Bool = True;
                 in y;